jekyll/_cci2/testing-ios.md

```diff
@@ -12,192 +12,82 @@ This document describes how to set up and customize testing for an iOS applicati
+ `10.1.0` | Xcode 10.1 (Build 10B61)        | 10.13.6 | [Installed software](https://circle-macos-docs.s3.amazonaws.com/image-manifest/v1901/index.html)
+ `10.0.0` | Xcode 10.0 (Build 10A255)       | 10.13.6 | [Installed software](https://circle-macos-docs.s3.amazonaws.com/image-manifest/v1893/index.html)
+ `9.4.1`  | Xcode 9.4.1 (Build 9F2000)      | 10.13.3 | [Installed software](https://circle-macos-docs.s3.amazonaws.com/image-manifest/v1881/index.html)
+ `9.3.1`  | Xcode 9.3.1 (Build 9E501)       | 10.13.3 | [Installed software](https://circle-macos-docs.s3.amazonaws.com/image-manifest/v1875/index.html)
+ `9.0.1`  | Xcode 9.0.1 (Build 9A1004)      | 10.12.6 | [Installed software](https://circle-macos-docs.s3.amazonaws.com/image-manifest/v1848/index.html)
 {: class="table table-striped"}
 
 ## Getting Started
 
-Select a macOS project you would like to build on the Add
-Projects page of the CircleCI application. **Note:** Changing build environment is no longer needed in 2.0. If your project is not listed as macOS, choose Linux project and then select macOS in the Operating System section.
+Select a macOS project repository you would like to build on the Add Projects page of the CircleCI application. You will need to ensure you have a [plan that allows macOS builds](https://circleci.com/pricing/), or if your project is open source, you can [apply for a special plan](https://circleci.com/open-source/) with free monthly build credits.
 
-## Basic Setup
+We highly recommend using [Fastlane](https://fastlane.tools) to build and sign your apps in CircleCI. Fastlane requires minimal configuration in most cases and simplifies the build-test-deploy process.
 
-After enabling macOS builds for your project, share
-the scheme that is going to be built on CircleCI so that CircleCI runs the
-correct build actions. Complete the following steps to share an existing scheme in Xcode:
+### Setting Up Your Xcode Project
 
-1. Choose Product > Scheme > Manage Schemes.
-2. Select the Shared option for the scheme to share, and click Close.
-3. Choose Source Control > Commit.
-4. Select the Shared Data folder.
-5. Enter your commit message in the text field.
-6. Select the "Push to remote" option (if your project is managed with Git).
-7. Click the Commit Files button.
-     A new `.xcscheme` file is located in the
-     `xcshareddata/xcschemes` folder under your Xcode project.
-8. Commit this file to your git repository so that CircleCI can access it.
+After setting up the project on CircleCI, you will need to ensure that the scheme you intend to build with Fastlane is marked as "shared" in your Xcode project. In most new projects created by Xcode, the default scheme will already by marked as "shared". To verify this, or to share an existing scheme, complete the following steps:
+
+1. Choose Product -> Scheme -> Manage Schemes
```

Comment:
Where do they do this?

Reply:
@rosieyohannan In Xcode, will clarify that